prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -1342,3 +1342,31 @@ periodics:
       - test-e2e-go-gke-ci
       - 'E2E_ARGS=--stress -run=TestStress*'
       - 'GCP_CLUSTER=stress-test'
+
+- name: vulnerability-scan
+  interval: 1h
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-misc
+    testgrid-tab-name: vulnerability-scan
+  cluster: build-kpt-config-sync
+  decorate: true
+  decoration_config:
+    timeout: 1h
```

Comment:
will it ever take an hour or is this a conservative timeout?

Reply:
It's a conservative timeout, it shouldn't take this long

Reply:
reduced to 30m